local/src/server/verifier-server.cc

```diff
@@ -593,7 +594,12 @@ TF_Serve_Connection(std::thread *t)
       break;
     }
 
-    errata = thread_info._session->accept();
+    // check for proxy protocol header from the socket
+    errata.note(S_DIAG, "Checking for proxy protocol header.");
+    auto pp_eratta = thread_info._session->read_and_parse_proxy_hdr();
+    errata.note(std::move(pp_eratta));
+    auto accept_errata = thread_info._session->accept();
```

Comment:
I'm concerned that this overhead call for all listening server connections will impact performance. It may be find, but have you tried this in prod sim to verify that we can still run traffic at a reasonable rate after this change?

Reply:
@lzx404243 ran this in prod sim and it seemed fine, near as we could tell. Only ran at 7krps, but pre-patch versions weren't running better.